packages/genesys-spark-components/src/components/legacy/gux-action-toast-legacy/example.html

```diff
@@ -1,4 +1,25 @@
-<h1>gux-action-toast</h1>
+<h1>gux-action-toast-legacy</h1>
+
+<div>
+  <gux-icon
+    style="color: #6c5619"
```

Comment:
Should we use a token here instead of hardcoding or is it ok to hardcode since it's a legacy component that will be deprecated soon?

Reply:
It should be fine to hardcode this, it's just in the example file